website/docs/addon_slack.md

```diff
@@ -15,83 +15,130 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-This module allows configuring profiles(when to trigger, for which combination of task, host and family)
-and templates(could contain {} placeholder) to send notification to Slack channel(s)
-whenever configured asset type is published.
+This addon enables you to set up profiles that specify when to trigger notifications, and for which combination of task, host, and product types.
+It also supports customizable message templates that use placeholders (denoted by curly brackets `{}`) that are automatically populated at the time of publishing.
 
 
 ## App installation
 
-Slack application must be installed to company's Slack first. 
+Please ensure that the Slack application is installed on the company's Slack before proceeding.
 
-Please locate `ayon/modules/slack/manifest.yml` file in deployed AYON installation and follow instruction at
-https://api.slack.com/reference/manifests#using and follow "Creating apps with manifests".
+<Tabs
+groupId="platforms"
+defaultValue="win"
+values={[
+{label: 'Windows', value: 'win'},
+{label: 'Linux', value: 'linux'},
+{label: 'Mac', value: 'mac'},
+]}>
+
+<TabItem value="win">
+
+1. Locate the `manifest.yml` file. You can find it in the `AppData` folder, typically at `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\slack_X.X.X\ayon_slack\`.
+2. To create your app, please follow the steps outlined in the "Creating apps with manifests" section at [Slack API Documentation](https://api.slack.com/reference/manifests#using).
+
+:::tip
+You can quickly navigate to the local app data using the environment variable shortcut: `%LOCALAPPDATA%`. For example, using `%LOCALAPPDATA%\Ynput\AYON\addons` as path to see your installed addons.
+:::
+
+</TabItem>
+
+<TabItem value="linux">
+
+1. Locate the `manifest.yml` file. You can find it in the user data folder, typically at `~/.local/share/Ynput/AYON/addons/slack_X.X.X/ayon_slack/`.
+2. To create your app, please follow the steps outlined in the "Creating apps with manifests" section at [Slack API Documentation](https://api.slack.com/reference/manifests#using).
+
+</TabItem>
+
+<TabItem value="mac">
+
+1. Locate the `manifest.yml` file. You can find it in the `Application Support` folder, typically at `~/Library/Application Support/Ynput/AYON/addons/slack_X.X.X/ayon_slack/`.
+2. To create your app, please follow the steps outlined in the "Creating apps with manifests" section at [Slack API Documentation](https://api.slack.com/reference/manifests#using).
+
+</TabItem>
+
+</Tabs>
 
 ### App icon
 
-If you would like to enrich bot with an icon, Slack admin must add the icon after app installation. 
+To give the bot a custom icon, the Slack admin needs to upload it following the app installation.
```

Comment:
Here I kind of have the same issue - "following the app installation" may be a bit confusing:

- Is that the Slack app installation using the manifest? I believe it's this.
- Is that the `ayon-slack` addon installation to AYON?

Would be great if we can find some wording that may bring less ambiguity.

Reply:
IMO, I think it's a matter of context. 
For instance, I assumed that the person who is reading this page and want to configure his/her/whatsoever slack settings is already aware of the app installation is related to a slack app.  Also, the instructions are referring to their slack website/instance.

Reply:
Let me know if this helps. Feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/218/commits/318a33c5df7ffdbad3b85e03b8ac5f4537262355